app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/DisplayImageActivity.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.HumanLearning2021.HumanLearningApp.view
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.TextView
+import com.github.HumanLearning2021.HumanLearningApp.R
+import com.github.HumanLearning2021.HumanLearningApp.model.CategorizedPicture
+import com.github.HumanLearning2021.HumanLearningApp.model.DummyCategorizedPicture
+
+class DisplayImageActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_display_image)
+
+        val image: DummyCategorizedPicture = intent.getSerializableExtra("display_image_image") as DummyCategorizedPicture
```

Comment:
Cast only down to `CategorizedPicture`, that way it will work with other implementations as well